### PR TITLE
Accessibility | Missing Form Labels

### PR DESCRIPTION
**Pull request by ChazUK**
Updated Questions to correctly use the **Label as Heading** pattern

Also fixed the external link icon being incorrectly placed when links wrap across two lines.

### CODE REVIEW

**Comment by ChazUK** [comment]
[Accessibility - Missing Form Labels](https://trello.com/c/jgzpL3Nu/159-accessibility-missing-form-labels)

File: src/app/features/workers/ethnicity/ethnicity.component.html [hunk 1]
```diff
@@ -1,34 +1,35 @@
 <app-error-summary *ngIf="submitted && form.invalid" [formErrorsMap]="formErrorsMap" [form]="form"></app-error-summary>
 
 <div class="govuk-grid-row">
```
Comment (omarCreativeDev) [inline src/app/features/workers/ethnicity/ethnicity.component.html]: maybe this class can go on the <form>?

File: src/app/features/workers/ethnicity/ethnicity.component.html [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
-  <div class="govuk-grid-column-two-thirds-from-desktop">
-    <h1 class="govuk-label-wrapper">
-      <label class="govuk-label govuk-label--xl govuk-!-margin-bottom-8" for="nationalInsuranceNumber">
-        What is their ethnicity?
-      </label>
-    </h1>
-  </div>
+  <form (ngSubmit)="onSubmit()" [formGroup]="form">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
```
Comment (omarCreativeDev) [inline src/app/features/workers/ethnicity/ethnicity.component.html]: how does `govuk-grid-column-two-thirds-from-desktop` differ from `govuk-grid-column-two-thirds`? I'm guessing its more mobile friendly?

Comment (ChazUK) [reply src/app/features/workers/ethnicity/ethnicity.component.html]: Yep, it has a responsive breakpoint

**Pull request merged**